assets/css/sensei-course-theme/quiz-compat.scss

```diff
@@ -15,5 +16,78 @@
 		.sensei-lms-quiz-back-to-lesson {
 			color: inherit;
 		}
+
+		input[type=file]::file-selector-button {
+			padding: 0.75rem 1.25rem;
+			color: inherit;
+		}
+	}
+}
+
+.wp-block-sensei-lms-question-answers {
+	& * {
+		font-size: 1.125rem;
+	}
+
+	input[type=file]::file-selector-button,
```

Comment:
Could we give this the same class name as the editor so that we only have to target 1 element?